src/Akka.Streams.Kafka/Internal/ConnectionChecker.cs

```diff
@@ -84,6 +84,7 @@ protected override void PreStart()
         {
             base.PreStart();
             StartTimer();
+            Become(Regular());
```

Comment:
Is this a bugfix?

Reply:
Yes, its a bug fix, discovered by the newly added ConnectionChecker unit tests